core/build.xml

```diff
@@ -39,13 +44,16 @@
 	   encoding="UTF-8"
 	   includeAntRuntime="false"
 	   debug="true"
-	   srcdir="src" destdir="bin"
+	   destdir="bin"
 	   classpath="apple.jar; 
                       library/jogl-all.jar; 
                       library/gluegen-rt.jar"
 	   nowarn="true">
       <!-- kinda gross, but if not using the JDT, this just ignored anyway -->
       <compilerclasspath path="${jdt.jar}; ../java/mode/jdtCompilerAdapter.jar" />
+      <src path="src"/>
+      <include name="processing/**"/>
+      <exclude name="processing/javafx/**" if="fx.missing"/>
```

Comment:
Let's call it `fx.unavailable`. And `if` can be used here? (Didn't know that...)